source/content/static-site-empty-upstream.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Use an Empty Upstream to Host a Static Site on Pantheon
+description: How to create a static site using an empty Upstream on Pantheon.
+contributors: [michellecolon-pantheon, kyletaylored]
+categories: [get-started]
+tags: [upstreams, site]
+reviewed: "2021-08-12"
+---
+
+Pantheon supports two of the largest open source content management systems (CMS) on the web, WordPress and Drupal. Occasionally, when customers move their portfolio of sites over to Pantheon, some sites are static and do not need a full CMS, and this is where empty Upstreams come in.
+
+The usual [Scope of Support](/support#scope-of-support) applies: Pantheon Support will help diagnose and remedy platform-related issues, but cannot resolve code-related issues.
+
+This process assumes you already have a static site developed and that you have access to the files. For more information about dependencies and which files to upload, consult the documentation for the static site generator you use.
+
+## What is an Empty Upstream?
+
+An empty Upstream is a [Custom Upstream](/custom-upstream) repository (a [Start State](/start-state) that does not have any content or code. For cases when a customer needs a blank start state, Pantheon provides an empty Upstream.
+
+Commonly, an empty Upstream is used as part of a [Build Tools](/guides/build-tools) setup. In this doc, we use it to host static HTML or PHP sites.
+
+## Create a Site With an Empty Upstream
+
+<Youtube src="PTVqXxwOJ4Q" title="Pantheon Static Site and Empty Upstreams" />
+
+There are two ways to create an empty Upstream site: via the [Pantheon Dashboard](/create-sites) and via [Terminus](/terminus).
+
+- Via the Pantheon Dashboard:
+
+  - Use the [Empty Site Upstream](https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65)
+
+- Via Terminus:
+
+  ```bash{promptUser: user}
+  terminus site:create my-new-site "My New Site" empty
+  ```
+
+## Upload the Static Site to Pantheon
+
+Now that you have static files and an empty Upstream that is ready to host them, add files to the Pantheon Site using [SFTP](/sftp) or [Git](/git).
+
+Once the files have been added to the Site, [commit the code](/sftp#committing-sftp-changes), and deploy the changes through Test to Live.
+
+## Frequently Asked Questions
+
+### Can I create a static site on Pantheon?
+
+Yes! You can start a new static site using an [empty Upstream](https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65).
+
+### Can I add static files to an existing WordPress or Drupal site?
+
+Yes, you can commit any static HTML file to your site that is unrelated to the CMS.
```

Comment:
@kyletaylored mentions that if you're doing this to add a static site within a CMS directory, that you should add the directory with its own index file - can we add a line about that to the Upload the Static Site to Pantheon section?